components/expenses/PayExpenseBtn.js

```diff
@@ -105,8 +110,17 @@ class PayExpenseBtn extends React.Component {
       }
       successMessage = intl.formatMessage(this.messages['paypalSuccessMessage']);
     } else if (get(expense, 'PayoutMethod.type') === 'BANK_ACCOUNT') {
-      selectedPayoutMethod = 'Transferwise';
+      selectedPayoutMethod = 'TransferWise';
       successMessage = intl.formatMessage(this.messages['transferwiseSuccessMessage']);
+      if (
+        host.plan.transferwisePayoutLimit !== null &&
+        host.plan.transferwisePayout >= host.plan.transferwisePayoutLimit
+      ) {
+        disabled = true;
+        disabledMessage = intl.formatMessage(this.messages['transferwisePlanLimitMessage'], {
+          a: (...chunks) => <a href={`/${host.slug}/edit/hostSettings`}>{chunks}</a>,
```

Comment:
Instead of <a> this needs to be an internal <Link so that we use client side navigation.

Alosi, I just renamed `hostSettings` to `host-plan`